src/containers/ScreenContainer.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import { StyleSheet, Text, View, Image, TouchableOpacity } from 'react-native';
+import PropTypes from "prop-types";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
import PropTypes from 'prop-types';
```